site/content/docs/5.3/utilities/overflow.md

```diff
@@ -92,8 +92,6 @@ Using Sass variables, you may customize the overflow utilities by changing the `
 
 ## CSS
 
-### Sass utilities API
-
-Overflow utilities are declared in our utilities API in `scss/_utilities.scss`. [Learn how to use the utilities API.]({{< docsref "/utilities/api#using-the-api" >}})
-
-{{< scss-docs name="utils-overflow" file="scss/_utilities.scss" >}}
+{{< sass-utilities-api-section title="Overflow" name="utils-overflow" file="scss/_utilities.scss" >}}
```

Comment:
There's an edge case I haven't seen before in `borders.md` where there are two calls to `scss-docs`:

```md
{{< scss-docs name="utils-borders" file="scss/_utilities.scss" >}}

{{< scss-docs name="utils-border-radius" file="scss/_utilities.scss" >}}
```

Let's perhaps wait for the discussion on the global approach to take place and not tackle it right away if it's too complicated, but here we might need to manage an array of names instead of a simple `name="utils-overflow"`.

Reply:
hh